Presentation.Web/app/components/data-processing/tabs/data-processing-registration-tab-main.view.html

```diff
@@ -254,10 +254,10 @@ <h4 data-element-type="dpaMainDetailHeader">{{vm.headerName}}</h4>
                     </thead>
                     <tbody>
                         <tr ng-repeat="dataProcessor in vm.dataProcessingRegistration.subDataProcessors" data-element-type="data-processing_sub-data-processors">
-                            <td>{{::dataProcessor.name }}</td>
-                            <td class="noRightBorder">{{::dataProcessor.cvrNumber }}</td>
+                            <td style="min-width: 300px">{{::dataProcessor.name }}</td>
```

Comment:
no no...
no inline styling like this.

Remember that TH actually have a width in `%` that you can set. The TH will then define the width of the columns in the rows